tests/libvmi/factory.go

```diff
@@ -50,6 +50,18 @@ func NewFedora(opts ...Option) *kvirtv1.VirtualMachineInstance {
 	return New(NamespaceTestDefault, RandName(DefaultVmiName), opts...)
 }
 
+// NewCirros instantiates a new CirrOS based VMI configuration,
+// without interfaces or networks.
```

Comment:
You mean with default ones, right?

Reply:
Idk... I mean I want the simplest possibly VMI spec that I update afterwards, suiting it to the use case of the test at hand.

Reply:
This is confusing, please take this line out.
There is no such thing "without interfaces or networks", these are implicit (default).
You can say just nothing, and that is it.

Reply:
Done.